infrastructure/terraform/bin/create-random-secrets.sh

```diff
@@ -6,4 +6,16 @@ function genpw () {
     openssl rand -base64 46 | cut -c1-"$1"
```

Comment:
Looks like the webhooks secret needs 66 characters, which is more than we support now. I must have overlooked that one when generating this number.

We should bump this to something like`-base64 94` which would work for up to 130 characters.